### PR TITLE
Add an announcement banner file for inclusion on the docs

### PR DESCRIPTION
**Pull request by Cadair**
This is a re-do of #604 

#630 didn't replace #604, the idea of this PR is to add a html file, which dynamically gets included in the docs via javascript (https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/announcements.html) this lets us by changing the content of this html file deploy an announcement banner to all versions of the astropy docs ([which have the config enabled](https://github.com/astropy/astropy/pull/16615)) without rebuilding them.

### CODE REVIEW

File: annoucement_banner.html [hunk 1]
```diff
@@ -0,0 +1 @@
+The <a href="https://ui.adsabs.harvard.edu/abs/2013A%26A...558A..33A/abstract">First Astropy Paper</a> recently <a href="https://www.astropy.org/paper1_10k_citation.html#astropy-paper-reaches-10-000-citations-article">hit 10,000 citations</a>. Thanks to everyone who has cited the paper and contributed to Astropy, <a href="https://www.astropy.org/contribute.html">Join Us</a> for the next 10,000!
```
Comment (pllim) [inline annoucement_banner.html:1]:
```suggestion
The <a href="https://ui.adsabs.harvard.edu/abs/2013A%26A...558A..33A/abstract">First Astropy Paper</a> recently <a href="https://www.astropy.org/paper1_10k_citation.html#astropy-paper-reaches-10-000-citations-article">hit 10,000 citations</a>. Thanks to everyone who has cited the paper and contributed to Astropy. <a href="https://www.astropy.org/contribute.html">Join Us</a> for the next 10,000!
```

**Comment by pllim** [comment]
Is this banner supposed to be temporary and be added to the docket of this issue?

* https://github.com/astropy/astropy.github.com/issues/631

**Comment by Cadair** [comment]
> Is this banner supposed to be temporary

yes

**Review by pllim — APPROVED** [review]
Sure why not. It is temporary anyway even if we end up not using it.

**Comment by Cadair** [comment]
To be clear, the *file* must always exist even if it's empty.

**Pull request merged**